.github/PR_TEMPLATE/PULL_REQUEST_CP_TEMPLATE.md

```diff
@@ -1,73 +1,50 @@
+This pull request is created by [automatic cherry pick workflow](https://github.com/flutter/flutter/wiki/Flutter-Cherrypick-Process#automatically-creates-a-cherry-pick-request)
+Please fill in the form below, and a flutter domain expert will evaluate this cherry pick request.
 
-**To fill in the form, you can edit this PR description and type your answers after the 'My Answer' keywords. <br>
-A flutter domain expert will evaluate this cherry pick request shortly after ALL questions are answered.**
+---
+#### Issue Link:
+What is the link to the issue this cherry-pick is addressing?<br>
 
-* Issue Link: What is the link to the issue this cherry-pick is addressing?<br>
+- [ ] My answer:
 
-<pre>
-  <b>My Answer:</b>
 
+---
+#### Changelog Description:
+Explain this cherry pick in one line that is accessible to most Flutter developers. See [best practices](https://github.com/flutter/flutter/wiki/Hotfix-Documentation-Best-Practices) for examples <br>
 
-</pre>
+- [ ] My answer:
 
-* Changelog Description: Explain this cherry pick in one line that is accessible to most Flutter developers
-        See https://github.com/flutter/flutter/wiki/Hotfix-Documentation-Best-Practices for examples (Bug fix, feature, docs update, ...)<br>
 
-<pre>
-  <b>My Answer:</b>
+---
+#### impact_description:
+What is the impact (ex. visual jank on Samsung phones, app crash, cannot ship an iOS app)? Does it impact development (ex. flutter doctor crashes when Android Studio is installed), or the shipping production app (the app crashes on launch)<br>
 
+- [ ] My answer:
 
-</pre>
+---
+#### Workaround:
+Is there a workaround for this issue?
 
-* Impacted Users: Approximately who will hit this issue (ex. all Flutter devs, Windows developers, all end-customers, apps using X framework feature)?<br>
+- [ ] My answer:
 
-<pre>
-  <b>My Answer:</b>
 
+---
+#### Risk:
+What is the risk level of this cherry-pick?
 
-</pre>
-
-* impact_description: What is the impact (ex. visual jank on Samsung phones, app crash, cannot ship an iOS app)? Does it impact development (ex. flutter doctor crashes when Android Studio is installed), or the shipping production app (the app crashes on launch)<br>
-
-<pre>
-  <b>My Answer:</b>
-
-
-</pre>
-
-* Workaround: Is there a workaround for this issue?<br>
-
-<pre>
-  <b>My Answer:</b>
-
-
-</pre>
-
-* Risk: What is the risk level of this cherry-pick?<br>
-
-<pre>
-  <b>My Answer:</b>
   - [ ] Low
   - [ ] Medium
   - [ ] High
+---
+#### Test Coverage:
+Are you confident that your fix is well-tested by automated tests?
 
-</pre>
-
-* Test Coverage: Are you confident that your fix is well-tested by automated tests?<br>
-
-<pre>
-  <b>My Answer:</b>
   - [ ] Yes
   - [ ] No
 
-</pre>
-
-* Validation Steps: What are the steps to validate that this fix works?<br>
-
-<pre>
-  <b>My Answer:</b>
+---
+#### Validation Steps:
+What are the steps to validate that this fix works?
 
+- [ ] My answer:
```

Comment:
Instead of using `- [ ] My answer:` would it be more readable to add generic text to be replaced?  The template leaves me to assume that these are checklist items which confuse me as a user.

Reply:
Sure, replaced with texts placeholders